src/pages/change-how-you-eat/index.js

```diff
@@ -95,7 +95,7 @@ const ChangeHowYouEatPage = () => (
       </div>
     </section>
     <section className={`${ChoiceStyles.how} ${Styles.how}`}>
-      <a name="how" />
+      <buttton id="#how" aria-label={"How"} >{null}</buttton>
```

Comment:
I believe that when the id's are created, they don't need a #, but when you reference them in a href, they do. As these buttons aren't working currently.